LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -289,14 +297,23 @@ def parse_config(data, me_url, mdsd_url, is_lad, az_resource_id, subscription_id
                     aggregator_str += " "*2 + "drop_original = true\n"
                     aggregator_str += " "*2 + "fieldpass = [" + non_ops_fields[:-2] + "]\n" #-2 to strip the last comma and space
                     aggregator_str += " "*2 + "stats = [\"mean\", \"max\", \"min\", \"sum\", \"count\"]\n\n"
+            
             elif is_vmi_rate_counter:
+                # Aggregator config for MDSD
                 aggregator_str += "[[aggregators.basicstats]]\n"
                 aggregator_str += " "*2 + "namepass = [\"" + plugin + "_mdsd\"]\n"
-                aggregator_str += " "*2 + "period = \"" + min_agg_period + "s\"\n"
+                aggregator_str += " "*2 + "period = \"" + min_interval + "\"\n"
```

Comment:
Replacing min_agg_period with min_interval seems correct if min_interval is influenced by configuration we receive, but I'm not sure if that's how min_interval is actually set - can you please confirm?

Reply:
This setting is only for VMI and their intervals are always set to 1 minute. but regardless whatever they set as their minimum interval, we will use that to aggregate metrics and poll them at half the interval time to make sure we meet their required time intervals